data/pampa_ncats/meta.yaml

```diff
@@ -0,0 +1,32 @@
+---
+name: pampa_ncats
+description: "PAMPA (parallel artificial membrane permeability assay) is a commonly\n        employed assay to evaluate drug permeability across the cellular\
+    \ membrane. PAMPA is a\n        non-cell-based, low-cost and high-throughput alternative to cellular models. Although\n        PAMPA does not model\
+    \ active and efflux transporters, it still provides permeability values\n        that are useful for absorption prediction because the majority of drugs\
+    \ are absorbed by\n        passive diffusion through the membrane."
+targets:
+    - id: permeability
+      description: Binary permeability in PAMPA assay.
+      units: Bool
+      type: categorical
+      names:
+          - binary permeability in PAMPA assay
+          - permeability in PAMPA assay
+          - PAMPA permeability
+identifiers:
+    - id: SMILES
+      type: SMILES
+      description: SMILES
+license: CC BY 4.0
+links:
+    - url: https://tdcommons.ai/single_pred_tasks/adme/#pampa-permeability-ncats
+      description: tdcommons.ai dataset url
+    - url: https://journals.sagepub.com/doi/full/10.1177/24725552211017520
+      description: corresponding publication
+num_points: 2034
+url: https://tdcommons.ai/single_pred_tasks/adme/#pampa-permeability-ncats
```

Comment:
same here. Let's decide on something and then we set the validation up accordingly